src/styles/zIndex.js

```diff
@@ -1,4 +1,5 @@
 export default {
+  snackbar: 900,
```

Comment:
I think snack bar should be even higher than popover, don't you think? I'd say `2900`, must still be less than toolbar.
Wouldn't you agree?